Microsoft.Azure.CosmosRepository/src/Providers/ICosmosContainerProvider.cs

```diff
@@ -1,23 +1,22 @@
-﻿// Copyright (c) IEvangelist. All rights reserved.
-// Licensed under the MIT License.
-
-using System.Threading.Tasks;
-using Microsoft.Azure.Cosmos;
-using Microsoft.Azure.CosmosRepository.Options;
+﻿// Copyright (c) IEvangelist. All rights reserved. Licensed under the MIT License.
 
 namespace Microsoft.Azure.CosmosRepository.Providers
 {
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Cosmos;
+    using Microsoft.Azure.CosmosRepository.Options;
+
     /// <summary>
-    /// The cosmos container provider exposes a means of providing
-    /// an instance to the configured <see cref="Container"/> object.
+    /// The cosmos container provider exposes a means of providing an instance to the configured
```

Comment:
Did you rely on some formatting tool, or something? Curious as I did not...

Reply:
Yes, I used the CodeMaid extension and built-in auto-format. It honors editorconfig and whatnot. My main objective was to get it into my standard form, which used to match what Microsoft does internally with their releases. Things have changed there a bit since I put out something there though so I'll have to ping some old friends and see what the newer standards are. I don't think they have guidance on code style across the company any more but I could be wrong.

Reply:
I work for Microsoft, the .NET team does have a common *.editorconfig* but each project can have their own thing. I was following what the Azure Cosmos DB .NET SDK repo was doing as it was my hope that they'd eventually adopt this project.